### PR TITLE
MMDP-359 Replace messages with toasts

### PR DESCRIPTION
**Pull request by charisschomba**
**What does this PR do?**

Toaster message should be used on add user page instead of the alert message

**Description of Task to be completed?**

- add toast message when creating a user
- add toast message when editing a user
- refactor tests affected
- fixed broken styles in user display
- fix full names col to display nothing if details are not updated.

**How should this be manually tested?**

- Fetch and checkout to `bug-add-toast-message-MMDP-359`
- Start server by running `yarn dev-sever`
- On your browser navigate to `http://localhost:1234/user`
- click `new user button`

**Expected results**

- After successfully adding a user a toast message to appear.
- The admin will be redirected to all users page, Fullname column should be empty
- If an error occurs a toast with error message should appear.

**What are the relevant Jira issues?**

[MMDP-359](https://viisaus.atlassian.net/secure/RapidBoard.jspa?rapidView=4&projectKey=MMDP&modal=detail&selectedIssue=MMDP-359)

**Screenshots (if appropriate)**

You can have screenshots where applicable
screenshot 2019-02-21 at 15 34 44
screenshot 2019-02-21 at 15 35 04
screenshot 2019-02-21 at 15 35 33

**Questions:**
N/A

### CODE REVIEW

File: client/assets/styles/Button/index.scss [hunk 1]
```diff
@@ -30,6 +30,9 @@ div.button__area {
   }
 }
 
+.thirteen.wide.column.actions-delete-edit {
+  margin-left: -1.75em
+
```
Comment (james-chege) [inline client/assets/styles/Button/index.scss]: Please close this style.

File: client/assets/styles/Button/index.scss [hunk 2]
```diff
@@ -30,6 +30,9 @@ div.button__area {
   }
 }
 
+.thirteen.wide.column.actions-delete-edit {
+  margin-left: -1.75em
```
Comment (james-chege) [inline client/assets/styles/Button/index.scss]: Replace this style property with `margin-left: -2em;` to fix the action buttons position.

File: client/assets/styles/Button/index.scss (hunk 1 above)
Comment (charisschomba) [reply client/assets/styles/Button/index.scss]: @james-chege Lemmi work on it

File: client/assets/styles/Button/index.scss (hunk 2 above)
Comment (charisschomba) [reply client/assets/styles/Button/index.scss]: on it

**Pull request merged**